cmd/run.go

```diff
@@ -38,13 +38,13 @@ func addRunSubcommands() {
 }
 
 func createSingleRunCmd(checkerType okgo.CheckerType, factory okgo.CheckerFactory) *cobra.Command {
-	checker, err := factory.NewChecker(checkerType, nil)
-	if err != nil {
-		panic(errors.Wrapf(err, "failed to create command for checker type %s", checkerType))
-	}
 	return &cobra.Command{
 		Use: string(checkerType),
 		Run: func(cmd *cobra.Command, args []string) {
+			checker, err := factory.NewChecker(checkerType, nil)
```

Comment:
I believe one of the reasons that we instantiated the checker before the command was to perform a check of sorts that "NewChecker" succeeds for every registered check, even without having to actually instantiate the command. Making this change to defer the instantiation does mean that, if `NewCheker` fails for some reason, we will only encounter that failure when trying to invoke it.

If we are observing performance issues then this trade-off is probably worth it (and you could argue that it's better if a check that doesn't instantiate doesn't just bring down the entire program), but wanted to flag.